app/src/main/java/com/isc/hermes/database/collection/Incidents.java

```diff
@@ -0,0 +1,55 @@
+package com.isc.hermes.database.collection;
+
+import com.isc.hermes.database.Hermesdb;
+import com.mongodb.client.MongoCollection;
+
+import org.bson.Document;
+
+/**
+ * This class that represents the collection of incidents from the hermes database.
+ */
+public class Incidents implements Collectible {
+
+    private Hermesdb hermesdb;
+    private MongoCollection<Document> incidents;
+    private static Collectible collection;
+    private final String COLLECTION_NAME = "incidents";;
```

Comment:
You have a ";" too much, it would be good if you delete it.

Reply:
done